Inputs/observables_list.yml

```diff
@@ -176,13 +176,13 @@ Observables:
      ifAbs: False
      ifJES: True
 
-    mass4lj_2p5:
-     bins: "|-2.0|110|180|220|300|400|600|3000|"
-     label: "m_{Hj} p_{T}_{j}>30 GeV |#eta_{j}|<4.7"
-     unit: "GeV"
-     gen: "GENmass4lj_2p5"
-     ifAbs: False
-     ifJES: True
+    #mass4lj_2p5:
+    # bins: "|-2.0|110|180|220|300|400|600|3000|"
+    # label: "m_{Hj} p_{T}_{j}>30 GeV |#eta_{j}|<4.7"
+    # unit: "GeV"
+    # gen: "GENmass4lj_2p5"
+    # ifAbs: False
+    # ifJES: True
```

Comment:
Currently, I just commented all observables with rapidity restricted to 2.5.

@vukasinmilosevic : Should I revert this back?